sql/core/src/main/scala/org/apache/spark/sql/execution/command/views.scala

```diff
@@ -198,10 +219,12 @@ case class CreateViewCommand(
 }
 
 /**
- * Alter a view with given query plan. If the view name contains database prefix, this command will
- * alter a permanent view matching the given name, or throw an exception if view not exist. Else,
- * this command will try to alter a temporary view first, if view not exist, try permanent view
- * next, if still not exist, throw an exception.
+ * Alter a view with given query plan.
+ *
+ * If the view name contains database prefix, this command will alter a permanent view matching the
```

Comment:
We may want separate command for alter global view.

It is not safe for a user to be able to alter a global view directly.